README.md

```diff
@@ -151,3 +151,83 @@ operation not supported
 ```
 This is probably because of a docker bug where, if you update kernel and don't reboot, docker gets confused. The fix is to reboot your system.
 More info: http://stackoverflow.com/questions/29546388/getting-an-operation-not-supported-error-when-trying-to-run-something-while-bu
+
```

Comment:
meta-resin should also cover non-managed OSes. given that, does it make sense to put here this stuff? And moreover, if we consider it makes sense, shall we put here attributes that hold secret/device specific data? Like apiKey, deviceApiKey etc?